doc/dev/fluxing.rst

```diff
@@ -0,0 +1,118 @@
+.. include:: ../include/links.rst
+
+.. _deimos_frames_report:
+
+Flux Calibration
+=================================
+
+Version History
+---------------
+
+=========   =============   =========== ===========
+*Version*   *Author*        *Date*      ``PypeIt``
+=========   =============   =========== ===========
+1.0         Joe Hennawi     31 Dec 2021 1.1.1
+=========   =============   =========== ===========
+
+----
+
+Basics
+------
+
+Testing :math:`\omega`: 3-D true position of the transient
```

Comment:
transient?

Reply:
This is a place holder until I have time to put the docs in there. Will happen before the PR goes through, but no need to wait for your review.